Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
```

Comment:
Might try and see if the slim version, eg `python:3.x-slim`, works. If it does that would save a ton of space.

Reply:
ideally we would use a multi-stage build, create wheels in the 'full fat' initial stage, and then copy them out + install in the 'slim' stage. I was unable to figure out how to do that with poetry, and trying it with pip introduced an issue with being unable to find the vendorized dep

Reply:
Yeah vendoring sucks. In the latest version I totally removed it so it should work with `pip` now.

Reply:
sweet i'll give it a pull